src/Model/Mail.php

```diff
@@ -117,14 +117,15 @@ public static function insert(array $msg, bool $notification = true)
 	/**
 	 * Send private message
 	 *
+	 * @param integer $sender_uid the user id of the sender, default 0
```

Comment:
It looks like having a valid user ID is necessary to send a message, so the default value here doesn’t make sense.
```suggestion
	 * @param integer $sender_uid the user id of the sender
```